src/test/java/core/basesyntax/service/FileWriterImplTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.impl.FileWriterImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterImplTest {
+    private static final String FILE_WRITE_PATH
+            = "src/test/java/core/basesyntax/resources/output.csv";
+    private static final String FILE_READ_PATH
+            = "src/test/java/core/basesyntax/resources/output.csv";
+    private static final String WRONG_PATH = "wrong/file.csv";

```

Comment:
you use this variable only once, make this variable local
```suggestion
    private static final String WRONG_PATH = "wrong/file.csv";

```